modules/graph/graph_test.go

```diff
@@ -0,0 +1,18 @@
+package graph_test
```

Comment:
This file should be contain some actual tests of what this PR is meant to accomplish.